docs/writing-tests/snapshot-testing.md

```diff
@@ -8,53 +8,173 @@ Storybook is a helpful tool for snapshot testing because every story is essentia
 
 ![Example Snapshot test](./snapshot-test.png)
 
-## Set up test runner
+## Migrating Tests
 
-Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
+The Storyshots addon was the original testing solution for Storybook, offering a highly extensible API and a wide range of configuration options for testing. However, it was difficult to set up and maintain, and it needed to be compatible with the latest version of Storybook, which introduced some significant architectural changes. As a result, Storyhots is now officially deprecated, is no longer being maintained, and will be removed in the next major release of Storybook. We recommend following the migration guide we've prepared to help you during this transition period.
```

Comment:
I feel like this section lost a bit of the essence of the [introduction here](https://github.com/storybookjs/test-runner/blob/next/MIGRATION.md), in a few ways:
- it doesn't mention the motivation that storyshots is incompatible with Storybook's high performance's on demand architecture (storyStoreV7)
- it doesn't mention that the migration can be done in different ways and the reader could choose. I guess this is tricky because the composeStories section is only rendered in react/vue, but maybe you can add a callout element that renders only for those renderers too?